### DISCUSSION TITLE
Criterion for a Feasible Solution

### DISCUSSION THREAD

**Discussion by Gwzwpxz** [ds4dm/ml4co-competition#7] [Q&A] [answered]
Hello,

We notice that there are both continuous and binary variables for some of the instances (e.g., the item placement problem). And we are wondering whether it is allowed to provide only the values of the binary variables or we have to give the whole solution for evaluation (since it seems non-trivial to directly conclude the values of the continuous variables via ML).

Best

**Comment by gasse** [comment, marked as answer]
Hello @Gwzwpxz ,

You can provide only values for the integer variables, and then SCIP will try to solve an LP to infer the value of the continuous ones. You can even provide values for a subset only if the integer variables, and SCIP will follow the same process to infer a value for the rest of the variables. In that case however, I suppose there are little chances that the remaining integer variables will be assigned an integer value.

Best,
